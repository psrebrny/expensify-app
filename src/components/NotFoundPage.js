import React from 'react';
import { Link } from 'react-router-dom';

export const NotFoundPage = () => {
  return (<div>404 - <Link to={'/'}>Go home</Link></div>);
};