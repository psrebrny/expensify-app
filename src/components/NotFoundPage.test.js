import React from 'react';
import { shallow } from 'enzyme';
import { NotFoundPage } from './NotFoundPage';

describe('components/notFoundPage.js', () => {
  test('render dashboard', () => {
    const wrapper = shallow(<NotFoundPage/>);
    expect(wrapper).toMatchSnapshot();
  });
});