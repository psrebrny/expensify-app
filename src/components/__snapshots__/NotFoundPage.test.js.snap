// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/notFoundPage.js render dashboard 1`] = `
<div>
  404 - 
  <Link
    replace={false}
    to="/"
  >
    Go home
  </Link>
</div>
`;
